service.py

```diff
@@ -0,0 +1,78 @@
+from concurrent import futures
+import grpc
+import time
+import utility
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+     def __init__(self):
+
+         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+             def __init__(self):
+                 pass
+
+             def GetStatus(self, request, context):
+                 print("Get Status")
+
+             def DownloadZippedFiles(self, request_iterator, context):
+                 if request_iterator.name:
+                     return utility.download_chunk(self.tmp_file_name)
+
+             def UploadFile(self, request_iterator, context):
+                 print("[INFO] Requesting UploadFile service")
+                 get_url=''
+                 for get_name in request_iterator:
+                     get_url = utility.upload_file_to_azure(request_iterator, get_name.file_name)
+
+                 status = hwsc_file_transaction_svc_pb2.FileTransactionResponse()
+                 for status in request_iterator:
+                     status.code = grpc.StatusCode.OK
+
+                 get_message = grpc.StatusCode.OK.name
+
+                 if get_url != '':
+                     return hwsc_file_transaction_svc_pb2.FileTransactionResponse(
+                     code=status.code,
+                     message=get_message,
+                     url=get_url
+                 )
+
+                 else:
+                     return hwsc_file_transaction_svc_pb2.FileTransactionResponse(
+                     code=grpc.StatusCode.ABORTED,

```

Comment:
`INTERNAL` is better suited for this in accordance with https://godoc.org/google.golang.org/grpc/codes